extensions/dataspace-authority-spi/src/main/java/org/eclipse/dataspaceconnector/registration/authority/model/Participant.java

```diff
@@ -32,10 +36,11 @@
  * Dataspace participant.
  */
 @JsonDeserialize(builder = Participant.Builder.class)
-public class Participant {
+public class Participant implements TraceCarrier {
 
     private String did;
     private ParticipantStatus status = ONBOARDING_INITIATED;
+    private Map<String, String> traceContext = new HashMap<>();
```

Comment:
Map.empty()

Reply:
you mean Map.of() ? 
Changed here and in the getter